packages/@react-spectrum/s2/test/SelectBoxGroup.test.tsx

```diff
@@ -66,44 +84,187 @@ function DisabledSelectBox() {
 }
 
 describe('SelectBoxGroup', () => {
+  let user;
+
+  beforeAll(function () {
+    jest.useFakeTimers();
+    jest.spyOn(window.HTMLElement.prototype, 'clientWidth', 'get').mockImplementation(() => 100);
+    jest.spyOn(window.HTMLElement.prototype, 'clientHeight', 'get').mockImplementation(() => 100);
+    jest.spyOn(window.HTMLElement.prototype, 'scrollHeight', 'get').mockImplementation(() => 50);
```

Comment:
why do we need to mock all these? this is usually for virtualised components, which this isn't